common/tools/dev-tool/launch.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const path = require("path");
+
+if (process.env.DEBUG) {
```

Comment:
seems to conflate how the `DEBUG` variable works, as it could be set to a value that disables logging for all but one particular channel. I have mixed feelings about using this without taking a dep on `debug` or using similar semantics to it.

Reply:
How about a `LOG_DEBUG` environment variable? I don't want to get too caught up in implementing the semantics of `debug`, when I think a simple toggle switch will do here. I could also add a `--debug` flag that is handled specially in the same way that `--help` is.